travis/update-release.py

```diff
@@ -79,16 +88,21 @@ def copyfile(src, dst):
         else:
             shutil.copy2(s, d)
 
-def get_container_images_data(r_stream, tag):
+def get_container_images_data(r_stream, search_stream, tag):
```

Comment:
`get_container_images_data` was being passed a particular release_stream (for e.g. 4.3.2.1), under each release_stream, there is a list comprising of 1 single '.z' release, 1 single release, multiple rc releases named as:
```
releases:
- release_streams:
     release_tag: 4.3.2.1
     - release_name: 4.3.2.1.z
     ...
     - release_name: 4.3.2.1
       released: true
     ...
     - release_name: 4.3.2.1.rc1
     ...
- release_streams:
     release_tag: 5.3.2.1
     ....
```
and we look for the stream whose name ends with '.z', expecting a release_stream to have all releases of only one release_tag.

additionally, for rc releases the search_stream should be `4.3.2.1.rcN` and for z it will be `4.3.2.1.z`, but the reason to not send search_stream with get_container_images_data was that we were always looking for the z release of a particular tagged release.